SyncIntegrationTests/xcodebuild.py

```diff
@@ -16,11 +16,33 @@ class XCodeBuild(object):
     testPlan = 'SyncIntegrationTestPlan'
     xcrun = XCRun()
 
-    def __init__(self, log):
+    def __init__(self, log, **kwargs):
+        self.scheme = kwargs.get("scheme", self.scheme)
+        self.testPlan = kwargs.get("test_plan", self.testPlan)
         self.log = log
 
-    def test(self, identifier):
-        self.xcrun.erase()
+    def install(self):
+        command = "find ~/Library/Developer/Xcode/DerivedData/Client-*/Build/Products/Fennec-* -type d -iname 'Client.app'"
```

Comment:
This path is the one I usually use. The path for `DerivedData` may differ in a CI environment. Where do the experiments run?

Reply:
Hmm, this will run on a Jenkins box, so we might have to update this later